packages/story-editor/tsconfig.json

```diff
@@ -18,15 +19,18 @@
     { "path": "../tracking" }
   ],
   "include": [
-    "src/app/api/*",
-    "src/app/config/*",
-    "src/app/currentUser/*",
-    "src/app/helpCenter/**/*",
+    "src/app/api",
+    "src/app/config",
+    "src/app/currentUser",
+    "src/app/helpCenter",
     "src/app/history",
+    "src/app/font",
     "src/app/story",
-    "src/types/*",
-    "src/constants/*",
+    "src/types",
+    "src/constants",
     "src/utils/*.ts",
     "src/typings/*",
+    "src/utils/usePreventWindowUnload.ts",
```

Comment:
These are covered by `"src/utils/*.ts",` above.

Reply:
Fixed in 9cffebe989,